package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "coverage": "nyc mocha",
     "start": "node index start",
-    "start-dev": "npm-run-all --parallel watch start",
+    "start-dev": "export NODE_ENV=\"dev\" && webpack-dashboard -- npm-run-all --parallel watch start",
```

Comment:
This is not cross platform.

Reply:
It looks to me like we don't even need that part, according to their README, you just add the plugin, and then you run the dashboard when and where you want, either per-command (when installed with `-g`) or in package.json like here.
That would be the easiest way to remove the issue, but @YaManicKill had specific reasons to do it through an env var and I'm not seeing it?